zlib-1.2.8/CMakeLists.txt

```diff
@@ -204,3 +204,16 @@ elseif(BUILD_SHARED_LIBS AND WIN32)
     # Creates zlib1.dll when building shared library version
     set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
 endif()
+
+add_library(ZLIB::header INTERFACE IMPORTED GLOBAL)
+if(CMAKE_GENERATOR MATCHES "Xcode")
```

Comment:
Humm, what is this for? It seems to break macOS and iOS builds (through XCode). It complains about not the impossibility of specifying -o when generating multiple outputs.